app/views/nouns/show.html.haml

```diff
@@ -5,6 +5,8 @@
     = @noun.name
 = title_with_actions(title) do
   = render "words/more_meanings", word: @noun
+  - if @noun.audio.attached?
```

Comment:
I believe you forgot the `FunctionWord`, which is also a `Word`.

Could we move that snippet into a partial, e.g. `words/audio` to be a bit more DRY?

Reply:
Oh good point, sorry didn't see that. I will extract it to a partial and also add it to functional word. Thanks!